clang/docs/ReleaseNotes.rst

```diff
@@ -504,6 +504,8 @@ AST Matchers
 - Fixed an ordering issue with the `hasOperands` matcher occuring when setting a
   binding in the first matcher and using it in the second matcher.
 
+- Fixed a crash when traverse lambda expr with invalid captures.
```

Comment:
Please mention the issue number in the release note (`#GH106444` and it will get auto-replaced with a URL to the issue).